src/ILLink.RoslynAnalyzer/RequiresAnalyzerBase.cs

```diff
@@ -355,6 +361,18 @@ protected bool IsMemberInRequiresScope (ISymbol containingSymbol)
 			return false;
 		}
 
+		/// <summary>
+		/// True if member of a call is considered to be annotated with the Requires... attribute. Primarily for property accessor declarations.
+		/// </summary>
```

Comment:
Maybe note that this method is like `IsMemberInRequiresScope`, except that it intentionally doesn't check the property annotations because the intended behavior is to require the base/override annotations to be on the same member.